### PR TITLE
Add bankaccount and link to prh

### PR DESCRIPTION
**Pull request by tswfi**
Add bank account information with a note that we accept donations
Add link to Information Service of the Finnish Register of Associations (PRH)

### CODE REVIEW

File: hacklab_ry.md [hunk 1]
```diff
@@ -37,3 +39,7 @@ Esimerkiksi
 > Yhdistyksen/hallituksen kokous kannatti [äänestyksessä] Hacklab ry.:n varsinaisen jäsenyyden hakemista (y-tunnus 3084371-1) sekä liiton sääntöjen ja tarkoituksen hyväksymistä.
 
 Mikäli yhdistyksen säännöt tai yhdistyksen kokouksen muu päätös ei toisin määrää, liittymispäätöksen voi tehdä yhdistyksen hallitus. Liiton suositus on tehdä päätös yhdistyksen kokouksessa. Jäsenyyden hakemisesta on syytä mainita kokouksen kutsussa ja tarjota linkki [liiton sääntöihin](/saannot.html), jotta osallistujilla on ollut riittävästi aikaa tutustua niihin ennen päätöstä.
+
+## Tilinumero
+
+IBAN-tilinumeromme on FI45 5670 0820 3659 12. Otamme vastaan lahjoituksia. Mikäli teet lahjoituksen, ilmoitathan siitä hallitukselle.
```
Comment (olmari) [inline hacklab_ry.md]: Figured an reference number for donations, `RF43 3800 1`. Also I'd change "ilmoitathan siitä hallitukselle" into some proper way of wishing (but not mandatory) "Toivomme ilmoitusta", all lawfully related data is in banking data, we can assume sender is donator if no other contact made.

File: hacklab_ry_en.md [hunk 2]
```diff
@@ -28,3 +31,7 @@ Purposes of the organisation briefly are:
  - be in charge of shared funds and resources
 
 [Complete official rules of the organisation only available in Finnish](/saannot.html)
+
+## Bank account
+
+IBAN-number FI45 5670 0820 3659 12. We accept donations. If you make a donation, please notify the board.
```
Comment (olmari) [inline hacklab_ry_en.md]: Same as above comment, but in english ;)

**Review by olmari — CHANGES_REQUESTED** [review]
My thoughts and added reference for donations.

File: hacklab_ry.md (hunk 1 above)
Comment (olmari) [reply hacklab_ry.md]: Also "anonymous donation" can be an thing, not that sender still has someones bank account :D but anyway I feel this wording is important in general, hence stalling for this =)

Comment (tswfi) [reply hacklab_ry.md]: changed the wording in 93081dc

File: hacklab_ry_en.md (hunk 2 above)
Comment (tswfi) [reply hacklab_ry_en.md]: changed the wording in 93081dc

**Review by olmari — APPROVED** [review]
LGTM

**Pull request merged**